packages/xo-server/src/xo-mixins/jobs/index.js

```diff
@@ -311,6 +330,12 @@ export default class Jobs {
           true
         )
 
+        type === 'backup' &&
+          app.emit('backup:postCall', {
+            ...data,
+            timestamp: Date.now(),
+          })
```

Comment:
IMHO, the payloads should be as close as possible to API calls: https://github.com/vatesfr/xen-orchestra/blob/ae2a92d229786791f3a249dbc5e689f30614a064/packages/xo-server/src/xo-mixins/api.js#L310-L328